engine/src/test/java/io/camunda/zeebe/engine/util/StreamProcessorRule.java

```diff
@@ -127,8 +127,8 @@ public StreamProcessorRule withEventApplierFactory(
     return this;
   }
 
-  public StreamProcessorRule withReplayMode(final ReplayMode replayMode) {
-    this.replayMode = replayMode;
+  public StreamProcessorRule withReplayMode(final StreamProcessorMode streamProcessorMode) {
```

Comment:
(same as above) 

Align the name of the method with the parameter name. We should change it to `streamProcessorMode()`.